src/domain.ts

```diff
@@ -62,13 +62,36 @@ interface DocMetaData {
   character: number;
 }
 
-interface TestMetaData {
+interface TestMetadataSpecification
+  extends TestMetadataAcceptedValues,
+    TestMetadataRelationships {
+  column_name: string;
+  model: string;
+}
+
+// for accepted_values
+interface TestMetadataAcceptedValues {
+  values?: string[];
+}
+
+// for relationship
+interface TestMetadataRelationships {
+  field?: string;
+  to?: string;
+}
```

Comment:
This is not a Union type but a single type.

Reply:
fixed